<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LocalControlPage</class>
 <widget class="QWidget" name="LocalControlPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>829</width>
    <height>697</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LocalControlPage</string>
  </property>
  <widget class="QPushButton" name="btnOpenRGB">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>530</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>RGB</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbRGB">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>530</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOpenAlarm">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>410</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>打开报警器</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOpenAlarm_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>440</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭报警器</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>661</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>500</y>
     <width>172</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QSlider" name="hsLight4">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnOpenLight4">
      <property name="text">
       <string>light4</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>440</y>
     <width>172</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QSlider" name="hsLight2">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnOpenLight2">
      <property name="text">
       <string>light2</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>410</y>
     <width>172</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QSlider" name="hsLight1">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnOpenLight1">
      <property name="text">
       <string>light1</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnOpenSteer">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>470</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>打开舵机</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCloseSteer">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>500</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭舵机</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOpenFan">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>410</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>打开风扇</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCloseFan">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>440</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭风扇</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOpenRelay1">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>470</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>打开继电器1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCloseRelay1">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>500</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭继电器1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOpenRelay2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>410</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>打开继电器2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCloseRelay2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>440</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭继电器2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOpenLock">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>470</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>打开电磁锁</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCloseLock">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>500</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭电磁锁</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>470</y>
     <width>172</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSlider" name="hsLight3">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnOpenLight3">
      <property name="text">
       <string>light3</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>231</width>
     <height>98</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="lbSerialPort">
        <property name="text">
         <string>SerialPort</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbSerialPortName"/>
      </item>
      <item>
       <widget class="QComboBox" name="cbSerialBandrate">
        <item>
         <property name="text">
          <string>9600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>19200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>57600</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSerialOpen">
        <property name="text">
         <string>打开</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
